src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/NativeLibraryHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon
+{
+    internal static class NativeLibraryHelper
+    {
+        private const string ConfigurationName =
+#if DEBUG
+            "Debug";
+#else
+            "Release";
+#endif
+
+        public static readonly Guid MonitorProfilerClsid = new Guid("6A494330-5848-4A23-9D87-0E57BBF6DE79");
+
+        public static string GetMonitorProfilerPath(string architecture) =>
+            GetSharedLibraryPath(architecture, "MonitorProfiler");
+
+        private static string GetSharedLibraryPath(string architecture, string rootName)
+        {
+            string artifactsBinPath = Path.GetFullPath(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "..", "..", ".."));
```

Comment:
I'm guessing there isn't a cleaner way to get this path? My only concern is that dropping back three levels feels somewhat fragile (as in if we move code around in the future this could break), but I don't actually have a suggestion for how to resolve this since you can't use an absolute path.

Reply:
Correct. but the relativeness between this assembly and the profiler assembly isn't likely going to change unless someone is moving the output folders around.